unattended-upgrade

```diff
@@ -1574,8 +1570,8 @@ def do_auto_remove(cache,             # type: apt.Cache
     else:
         for pkgname in auto_removable:
             cache[pkgname].mark_delete()
-        if is_autoremove_valid(cache, "", auto_removable, blacklisted_pkgs,
-                               whitelisted_pkgs):
+        if is_autoremove_valid(cache, "", auto_removable, blacklist,
```

Comment:
This can probably now be a single line (probably <79 chars now)?

Reply:
It would become == 80, but tried that, too :-)